docs/pageobjects.md

```diff
@@ -185,7 +185,7 @@ class AttachFile {
 }
 
 // For inheritance
-module.exports = new AttachFile()
+module.exports = new AttachFile();	
 module.exports.AttachFile = AttachFile
```

Comment:
Why do-comma not places here like prev line?

Reply:
The current file on master didn't have it on both line. I'll add them to both lines.
https://github.com/Codeception/CodeceptJS/commit/634801483de24179ce69312ca58d2cc4b4785b83